router/router.go

```diff
@@ -96,6 +98,18 @@ func (r *Router) InjectPacketFilter(fn FilterFn) {
 	})
 }
 
+func (r *Router) EnableWakeupBroadcasts() {
+	r.state.Act(r.state, func() {
+		r.state._sendBroadcastIn(0)
+	})
+}
+
+func (r *Router) DisableWakeupBroadcasts() {
+	r.state.Act(r.state, func() {
+		r.state._broadcastTimer.Stop()
```

Comment:
You may need to drain the timer channel as well, see https://pkg.go.dev/time#Timer.Stop